docs/src/examples/gs2-example.adoc

```diff
@@ -1,10 +1,15 @@
+:lang: en
+:toc:
+
```

Comment:
Shouldn't we agree on standardized header tags and add them to all files ?

Reply:
The :lang: likely is undisputed in every file. The table of contents maybe not. I did not care too much - the focus is on getting the .po files autogenerated to be fed into weblate. Everything else will then be autogenerated from the English.